CHANGELOG.md

```diff
@@ -8,15 +8,17 @@ Bug fixes:
 
 * Fix error message when the method name is not a Symbol or String for `Kernel#respond_to?` (#2132, @ssnickolay)
 * Fixed setting of special variables in enumerators and enumerables (#1484).
+* Fixed return value of `Enumerable#count` and `Enumerable#uniq` with multiple yielded arguments (#2145, @LillianZ).
 
 Compatibility:
 
 * Implement `String#undump` (#2131, @kustosz)
 * `Errno` constants with the same `errno` number are now the same class.
 * Implement `Enumerable#tally` and `Enumerable#filter_map` (#2144 and #2152, @LillianZ).
+* Pass more `Enumerator::Lazy#uniq` and `Enumerator::Lazy#chunk` specs.  (#2146, @LillianZ).
 
 Performance:
-
+* Refactor and implement more performant `MatchData#length` and `Array#sample`. (#2147 and #2148, @LillianZ).
```

Comment:
Could you make two separate entries for those? They seem unrelated.
Also there should be a blank line between `Performance:` and the first item for consistency.